Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.44250&attempt_id=aa0e71ff-1a3a-4e20-91e4-2a0782b9c141)

Attempted: strelka

<details><summary>Partial log (click to expand)</summary><p>

```
[  0%] Built target strelka_buildtime_config
Scanning dependencies of target strelka_cxx_buildtime_config
[  0%] Built target strelka_cxx_buildtime_config
Scanning dependencies of target strelka_bgzfx
[  0%] Building bgzf_extras
make[2]: *** [redist/CMakeFiles/strelka_bgzfx.dir/build.make:63: redist/bgzf_extras-v0.1/bgzf_cat] Error 2
make[1]: *** [CMakeFiles/Makefile2:209: redist/CMakeFiles/strelka_bgzfx.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
builder for '/nix/store/601swdzp9i310r4kkpshx4dh0x574bdw-strelka-2.9.5.drv' failed with exit code 2
error: build of '/nix/store/601swdzp9i310r4kkpshx4dh0x574bdw-strelka-2.9.5.drv' failed
```
</p></details>

--- a/pkgs/applications/science/biology/strelka/default.nix
+++ b/pkgs/applications/science/biology/strelka/default.nix
@@ -1,0 +1,37 @@
+{stdenv, fetchFromGitHub, cmake, zlib, python2}:
+
+stdenv.mkDerivation rec {
+  name = "strelka-${version}";
+  version = "2.9.5";
+
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = "strelka";
+    rev = "v${version}";
+    sha256 = "0x4a6nkx1jnyag9svghsdjz1fz6q7qx5pn77wphdfnk81f9yspf8";
+  };
+
+  buildInputs = [ cmake zlib python2 ];
+
+  preConfigure = ''
+    sed -i 's|/usr/bin/env python|${python2}/bin/python|' src/python/lib/makeRunScript.py
+    patchShebangs .
+  '';
+
+  postFixup = ''
+    pushd $out/lib/python/pyflow
+    sed -i 's|/bin/bash|${stdenv.shell}|' pyflowTaskWrapper.py
+    rm pyflowTaskWrapper.pyc
+    echo "import pyflowTaskWrapper" | python2
+    popd
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Germline and small variant caller";
+    license = licenses.gpl3;
+    homepage = https://github.com/Illumina/strelka;
+    maintainers = with maintainers; [ jbedo ];
+    platforms = [ "x86_64-linux" ];
+  };
+
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -20393,6 +20393,8 @@ with pkgs;
 
   star = callPackage ../applications/science/biology/star { };
 
+  strelka = callPackage ../applications/science/biology/strelka { };
+
   varscan = callPackage ../applications/science/biology/varscan { };
 
   hmmer = callPackage ../applications/science/biology/hmmer { };

